### PR TITLE
feat: fixed percentages :octocat:

### PR DESCRIPTION
**Pull request by Mcnoble1**

### CODE REVIEW

File: expenses-chart-component-main/index.js [hunk 1]
```diff
@@ -48,7 +48,10 @@ function getCharts(display) {
     const getBars = document.querySelectorAll('.bars')
     getBars.forEach(element =>{
         element.addEventListener('mouseover', () => {
-            show()
+            show(element);
```
Comment (KemiDoyin) [inline expenses-chart-component-main/index.js:51]: I think I get this now.

**Comment by KemiDoyin** [comment]
Thank you👍 
TCLG.

**Comment by Mcnoble1** [comment]
> Thank you👍 TCLG.

What's TCLG

File: expenses-chart-component-main/index.js (hunk 1 above)
Comment (Mcnoble1) [reply expenses-chart-component-main/index.js:51]: Awesome. That's my girl.

**Comment by KemiDoyin** [comment]
This code looks good

**Pull request merged**